configure.ac

```diff
@@ -2384,6 +2384,28 @@ CheckOpenGLX11()
     fi
 }
 
+dnl Find KMSDRM OpenGL (GLVND, NOT GLX and it's huge dependencies.)
+CheckOpenGLKMSDRM()
+{
+    if test x$enable_video = xyes -a x$enable_video_opengl = xyes -a x$enable_video_kmsdrm = xyes; then
+        AC_MSG_CHECKING(for OpenGL (GLVND) support)
+        video_opengl=no
+        AC_TRY_COMPILE([
+         #include <GL/gl.h>
+        ],[  
+        ],[  
+        video_opengl=yes
+        ])   
+        AC_MSG_RESULT($video_opengl)
+        if test x$video_opengl = xyes; then 
+            AC_DEFINE(SDL_VIDEO_OPENGL, 1, [ ]) 
+            AC_DEFINE(SDL_VIDEO_OPENGL_GLVND, 1, [ ]) 
```

Comment:
The new `SDL_VIDEO_OPENGL_GLVND` is not referenced anywhere, nor is it added to SDL_config.h.in

Reply:
Removed as it wasn't being used, sorry.